assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -524,13 +524,21 @@ che.workspace.plugin_registry_url=https://che-plugin-registry.prod-preview.opens
 # In case Che plugins tooling is not needed value 'NULL' should be used
 che.workspace.devfile_registry_url=https://che-devfile-registry.prod-preview.openshift.io/
 
-# Defines a default value for persist volumes that clients like Dashboard
-# should propose for users during workspace creation.
-# Possible values: true or false
-# In case of true - PersistentVolumeClaims are used by declared volumes by user and plugins. `true`
-# value is supposed not to be set explicitly in Devfile attributes since it's default fixed behaviour.
-# In case of false - emptyDir is used instead of PVCs. Note that data will be lost after workspace restart.
-che.workspace.persist_volumes.default=true
+# The configuration property that defines available values for storage types that clients like
+# Dashboard should propose for users during workspace creation/update.
+# Available values:
+#   - 'persistent': Persistent Storage slow I/O but persistent.
+#   - 'ephemeral': Ephemeral Storage allows for faster I/O but may have limited storage
+#       and is not persistent.
+#   - 'async': Experimental feature: Asynchronous storage is combination of Ephemeral
+#       and Persistent storage. Allows for faster I/O and keep your changes, will backup on stop
+#       and restore on start workspace.
+che.workspace.storage.available_types=persistent,ephemeral,async
+
+# The configuration property that defines a default value for storage type that clients like
+# Dashboard should propose for users during workspace creation/update.
+# The 'async' value not recommended as default type since it's experimental
+che.workspace.storage.default_type=persistent
```

Comment:
should we call it `che.workspace.storage.prefered_type=persistent` wdyt?

Reply:
Both works for me)

Reply:
sound ok